pkgs/by-name/ju/jumper/package.nix

```diff
@@ -0,0 +1,41 @@
+{
+  stdenv,
+  fetchFromGitHub,
+  pkgs,
+  lib,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "jumper";
+  version = "0-unstable-2024-08-26";
+  src = fetchFromGitHub {
+    owner = "homerours";
+    repo = "jumper";
+    rev = "264f291754a99ea4aa44d05e6a1a0f4b9159fc8c";
+    hash = "sha256-6++r0mBrV+KhQc7WRiKd8l52CKJUcvzPFqe4S6O6J84=";
+  };
+
+  nativeBuildInputs = [
+    pkgs.which # removes comp warning
```

Comment:
Just like with `fetchFromGitHub`, `which` can be put as an entry into the top attrset and then it can be used without `pkgs.`

The same is true for `fzf`